charts/teuto-cnpg/templates/objectstore.yaml

```
@@ -0,0 +1,21 @@
+{{- if .Values.backup }}
+apiVersion: barmancloud.cnpg.io/v1
+kind: ObjectStore
+metadata:
+  name: {{ printf "%s-objectstore" .Release.Name }}
```

Yes, this is an objectstore...

```suggestion
  name: {{ .Release.Name }}
```